### DISCUSSION TITLE
"img" shortcode doesn't work

### DISCUSSION THREAD

**Discussion by luispaulomr** [thegeeklab/hugo-geekblog#555] [Q&A] [answered]
I can't get the "img" shortcode to diplay my images, and I don't get any build errors/warnings.

What I've tried:
- multiple paths (absolute, relative) -> didn't work
- using markdown and html for displaying images -> this work
- using hugo version 0.93 (version shown on this repo's README) -> didn't work

Also, I see in the exampleSite there are two related files:
- layouts/shortcodes/sprites.html
- data/properties/shortcode-images.yaml

Should I have them in my project? Anyway, I tried this and didn't work too.

Thanks

**Comment by xoxys** [comment]
Not sure what you tried already but the usage of the `img` shortcode is documented in [Advanced Images](https://hugo-geekblog.geekdocs.de/posts/post-with-images/). The full example (used for the documentation) can be found at https://github.com/thegeeklab/hugo-geekblog/tree/main/exampleSite/content/posts/post-with-images

**Comment by luispaulomr** [comment]
Yes, that's the guide I followed.

I have this file structure:

```
post/
│   post.md
└───images/
     └───first.png
```

My header looks like this:

```
resources:
    - name: first
      src: "images/first.png"
```

And I use the shortcode as follows:

```
<!-- spellchecker-disable -->                                                    
{{< img name="first" >}}                                                         
<!-- spellchecker-enable -->   
```

But the image is not being displayed. Am I missing something?

**Reply by xoxys** [reply, marked as answer]
The documentation links https://gohugo.io/content-management/page-resources/

> Page resources are only accessible from [page bundles](https://gohugo.io/content-management/page-bundles/), those directories with index.md or _index.md files at their root. Page resources are only available to the page with which they are bundled.

The recommended structure would be (also see https://github.com/thegeeklab/hugo-geekblog/tree/main/exampleSite/content/posts/post-with-images):

```
posts/
└───my-post/
    └───index.md
    └───images/
        └───first.png
```

**Reply by luispaulomr** [reply]
Ok, it worked by changing the file name to "index.md".

Thanks for the help!
